data/korath/korath missions.txt

```diff
@@ -257,3 +257,548 @@ mission "Small Scale Delivery"
+			# Iris's explanation is extremely biased because her primary source
+			# is the Korath exiles. She is presenting a human interpretation of
+			# the Exiles' feelings about their own fate.
+			`	"The imperialist Drak race sent living weapons, the Archons, to conquer the Korath under the guise of protection. The Archons' antimatter weapons and seeming invincibility was too much, until the Korath gathered a larger fleet. When they managed to injure one Archon, the others revealed frightening powers. They tore open rifts in reality, simply swallowing up Korath ships. Hidden super-weapons in planetoids, intended as mines, were harmless, as the Archons shattered the planetoids and atomized the weapons. In every battle, all Korath military ships were destroyed, but the civilians were somehow unscathed.`
+			`	"The Korath saw certain defeat, and in surrender the Drak offered two options. The Korath may give up their religions and live as pets of their former friends, the Quarg. Or, they could keep their religions and live in starving exile near us.`
+			`	"The Drak left the exiles with a warning: destroy Far'en Lei, and we will kill you all. You may never leave exile until you heal that planet and live there peacefully. As a precaution, and almost as a religion, they forbid weapons on that planet. A condition YOU violated." She looks heated for a moment before continuing.`
+			`	"As to their guardianship over us, it violates the letter of the law, but not the intent. The Korath believe the Drak will look favorably upon them for protecting the weak. Along with defending us against major attacks, they have provided extensive assistance. Their aid - at our insistence - has been purely civilian: Anything we receive, the Syndicate may somehow obtain, and I shudder to think of what they may do with the wrong knowledge."`
+				goto check
+
+			label compassion
+			`	"Now that I have satisfied your curiosity, I have a request for you."`
+			choice
+				`	"Of course. What is it?"`
+				`	"I'll hear it."`
+			`	"I sense you have a part to play on the galactic stage, which I should not forbid. The rest of the galaxy knows our protectors as monsters; mindless destroyers to be hunted for thrill or profit. All I ask is that you act with compassion and understanding."`
+			choice
+				`	"You've made a convincing case. I'll help them find their place in the galaxy."`
+				`	"Compassion for the Korath? That's asking a lot, but I'll try."`
+				`	"We will have to agree to disagree, but I will not forget your words."`
+					goto nosee
+			`	She closes her eyes and sighs happily. "Thank you. We do not share this information lightly, and I'm glad we judged well." Opening her eyes and smiling, she does a slight bow and departs.`
+				decline
```

Comment:
Accept or +rep or set a flag or something to keep track of the response for future missions.  Same for the Far'en Lai mission.
```suggestion
				accept
```

Reply:
It turns out that missions which have to automatically fail cannot have an "accept" in their dialog, or things don't work as intended. Instead, I added two variables:

- `Korath happiness with first contact` = 1 or 2
- `willingness to agree to Iris's request to help Korath` = 1, 2, or 3